error: cgo argument has Go pointer to Go pointer with LONG VARCHAR column in Linux

Note: This error doesn't happen in Windows.

- os Version: Ubuntu 18.04.1
- cli-driver Version: 11.1.4.4
- Db2 Server Version: 11.5.0.1077
- go Version: 1.13.5
<details><summary><code>go env</code> Output:</summary><br><pre>
$ go env
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/sdelacruz/.cache/go-build"
GOENV="/home/sdelacruz/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/home/sdelacruz/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/snap/go/4901"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/snap/go/4901/pkg/tool/linux_amd64"
GCCGO="gccgo"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/home/sdelacruz/Desktop/x/sdi/go.mod"
CGO_CFLAGS="-I/home/sdelacruz/Desktop/sdi/clidriver/include"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-L/home/sdelacruz/Desktop/sdi/clidriver/lib"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build863051176=/tmp/go-build -gno-record-gcc-switches"
</pre></details>

Steps to Reproduce:

1. Use commit 5266ca44206f
2. Try to do a select query with two or more columns where a column is `LONG VARCHAR`. If last column is `LONG VARCHAR` then no error. But when there is a column after the `LONG VARCHAR` column then the program panic with this error:

```
panic: runtime error: cgo argument has Go pointer to Go pointer

goroutine 19 [running]:
github.com/ibmdb/go_ibm_db/api.SQLGetData.func1(0x4000300010001, 0xc00006e138, 0xc000000004, 0xc00006e118, 0xc00029fe48)
	/home/sdelacruz/go/pkg/mod/github.com/ibmdb/go_ibm_db@v0.2.1-0.20200225090006-5266ca44206f/api/zapi_unix.go:83 +0x4c
github.com/ibmdb/go_ibm_db/api.SQLGetData(0x4000300010001, 0xc00006e138, 0x4, 0xc00006e118, 0xc00000e120)
	/home/sdelacruz/go/pkg/mod/github.com/ibmdb/go_ibm_db@v0.2.1-0.20200225090006-5266ca44206f/api/zapi_unix.go:83 +0x59
github.com/ibmdb/go_ibm_db.(*BufferLen).GetData(...)
	/home/sdelacruz/go/pkg/mod/github.com/ibmdb/go_ibm_db@v0.2.1-0.20200225090006-5266ca44206f/column.go:25
github.com/ibmdb/go_ibm_db.(*BindableColumn).Value(0xc00006e100, 0xc000010001, 0x2, 0xc13a40, 0xc00000e120, 0x0, 0x0)
	/home/sdelacruz/go/pkg/mod/github.com/ibmdb/go_ibm_db@v0.2.1-0.20200225090006-5266ca44206f/column.go:268 +0x22e
github.com/ibmdb/go_ibm_db.(*Rows).Next(0xc000010018, 0xc0003de1b0, 0x3, 0x3, 0x203000, 0x2)
	/home/sdelacruz/go/pkg/mod/github.com/ibmdb/go_ibm_db@v0.2.1-0.20200225090006-5266ca44206f/rows.go:99 +0xcb
database/sql.(*Rows).nextLocked(0xc0003e4000, 0x430000)
	/snap/go/4901/src/database/sql/sql.go:2767 +0xd5
database/sql.(*Rows).Next.func1()
	/snap/go/4901/src/database/sql/sql.go:2745 +0x3c
database/sql.withLock(0xe3f2a0, 0xc0003e4030, 0xc00029ffe0)
	/snap/go/4901/src/database/sql/sql.go:3184 +0x6d
database/sql.(*Rows).Next(0xc0003e4000, 0x3)
	/snap/go/4901/src/database/sql/sql.go:2744 +0x87
.........
exit status 2
```

--- a/column.go
+++ b/column.go
@@ -101,9 +101,9 @@ func NewColumn(h api.SQLHSTMT, idx int) (Column, error) {
 	case api.SQL_BINARY, api.SQL_VARBINARY, api.SQL_BLOB:
 		return NewVariableWidthColumn(b, api.SQL_C_BINARY, size), nil
 	case api.SQL_LONGVARCHAR:
-		return NewVariableWidthColumn(b, api.SQL_C_CHAR, 0), nil
+		return NewVariableWidthColumn(b, api.SQL_C_CHAR, size), nil
 	case api.SQL_WLONGVARCHAR, api.SQL_SS_XML:
-		return NewVariableWidthColumn(b, api.SQL_C_WCHAR, 0), nil
+		return NewVariableWidthColumn(b, api.SQL_C_WCHAR, size), nil
 	case api.SQL_LONGVARBINARY:
 		return NewVariableWidthColumn(b, api.SQL_C_BINARY, 0), nil
 	case api.SQL_DBCLOB:

